packages/grafana-ui/src/components/Stack/Stack.internal.story.tsx

```diff
@@ -31,4 +33,176 @@ export const Basic: StoryFn<typeof Stack> = ({ direction = 'column', gap = 2 })
   );
 };
 
+export const TestCases: StoryFn<typeof Stack> = ({ direction = 'column', gap = 2 }) => {
```

Comment:
I think we will want to disable controls for this story since they don't do anything